linked_list/linked_list.py

```diff
@@ -0,0 +1,253 @@
+class Node(object):
+    def __init__(self, data=None, next_=None):
+        # Initialises Node
+        self.data = data
+        self.next = next_
+
+    def __str__(self):
+        return str(self.data) if self.next is None \
+            else '{0} -> {1}'.format(self.data, self.next)
+
+
+class Linked_list(object):
```

Comment:
The name should be LinkedList (ProperCase). 
I wonder why pep8 is not caching these issues.